tools/topology/topology2/include/platform/intel/bt-generic.conf

```diff
@@ -0,0 +1,158 @@
+#
+# Bluetooth Audio Offload support
+#
+Object.Dai {
+	SSP.2 {
+		#id 8
+		#default_hw_conf_id 8
+		name SSP2-BT
+		id @SSP2_ID@
+		default_hw_conf_id @SSP2_ID@
+		sample_bits 16
+		Object.Base.data.'ssp_data1' {
+			bclk_delay 0
+			clks_control 0
+			quirks "lbm_mode"
+			frame_pulse_width 0
+			mclk_id 0
+			tdm_padding_per_slot false
+		}
+		Object.Base.data.'ssp_data2' {
+			bclk_delay 0
+			clks_control 0
+			quirks "lbm_mode"
+			frame_pulse_width 0
+			mclk_id 0
+			tdm_padding_per_slot false
+		}
+		Object.Base.data.'ssp_data3' {
+			bclk_delay 0
+			clks_control 0
+			quirks "lbm_mode"
+			frame_pulse_width 0
+			mclk_id 0
+			tdm_padding_per_slot false
+		}
+		Object.Base.hw_config."hw_config1" {
```

Comment:
Rather than saying "hw_config1" can we be more meaningful e.g. "ssp_16k_dspa". This would be easier to track in the code.